include/aie/Dialect/AIE/IR/AIE.td

```diff
@@ -1205,6 +1205,22 @@ def AIE_ObjectFifoCreateOp: AIE_Op<"objectFifo.createObjectFifo", []> {
   }];
 
   let extraClassDeclaration = [{
+    bool hasName() {
+      if(auto attr = getOperation()->getAttrOfType<StringAttr>(SymbolTable::getSymbolAttrName())) {
+        return true;
+      } else {
+        return false;
+      }
+    }
+    StringAttr name() {
+      if(auto attr = getOperation()->getAttrOfType<StringAttr>(SymbolTable::getSymbolAttrName())) {
+        return attr;
+      } else {
+        emitOpError("does not have '") << SymbolTable::getSymbolAttrName() <<
+          "' attribute specified";
+      }
+      llvm_unreachable("unreachable");
+    }
```

Comment:
Can we have one function returning `Optional<StringAttr>`?  Instead of aborting the program when there's no name can we emit an error message where the error happens?  e.g. "error: could not perform X because Y is un-named"